src/Game.java

```diff
@@ -4,95 +4,94 @@
 import java.util.Random;
 
 public class Game {
-    boolean whose_turn;
-    public boolean is_running;
+    boolean whoseTurn;
```

Comment:
Tutaj zmienił bym nazwę, bo na pierwszy rzut oka jeżeli to będzie true to nie wiem czy to X czy O zaczyna